docs/users/installation.md

```diff
@@ -75,6 +75,22 @@ following settings to `build.sbt`
  )
 ```
 
+> To enable SemanticDB related settings for all projects in your build.sbt, 
```

Comment:
I think it would be good to recommend this as the default installation method and document the other approach for users on sbt <v1.3. What do you think?

Reply:
I'm a bit worrying that ThisBuild settings introduce SemanticDB settings across the projects, some of which may not require SemanticDB but incur overhead in compilation time.

Disclaimer: I have not faced the case the overhead is critical. Do you think it would be enough minor?

Reply:
In case ThisBuild setting becomes as default, it would be better to modify an error message to ThisBuild scoped settings. 

https://github.com/scalacenter/sbt-scalafix/blob/ae173f44cf4875e9b1d4a172d9ce7d76e2e7076e/src/main/scala/scalafix/sbt/ScalafixPlugin.scala#L202-L203

Reply:
I agree, we can do that separately

Reply:
Okay, I can do it.
I'll update this PR with mentioning about this risk as well.

> I'm a bit worrying that ThisBuild settings introduce SemanticDB settings across the projects, some of which may not require SemanticDB but incur overhead in compilation time.

Reply:
I've sent a PullRequest to change the error message.
https://github.com/scalacenter/sbt-scalafix/pull/90